File: Tarea1Unidad3Java/src/Productos/Alimento.java
package Productos;

import java.time.LocalDate;
import Productos.helpers.tipoProducto;

public class Alimento extends Producto {
    private LocalDate fechaVencimiento;

    public Alimento(String nombre, double precio, int stock, String fechaImportacion, String numSerie, LocalDate fechaVencimiento) {
        super(nombre, tipoProducto.ALIMENTO, precio, stock, fechaImportacion, numSerie);
        this.fechaVencimiento = fechaVencimiento;
    }

    public LocalDate getFechaVencimiento() {
        return fechaVencimiento;
    }

    @Override
    public String getDatos(){
        return String.format("%s    Fecha de Caducidad: %s", super.getDatos(), getFechaVencimiento());
    }
}

File: Tarea1Unidad3Python/Alimento.py
from datetime import datetime
from enum import Enum
from Producto import Producto, tipoProducto

class Alimento(Producto):
    def __init__(self, nombre, precio, stock, fecha_importacion, fecha_vencimiento):
        super().__init__(nombre, tipoProducto.ALIMENTO, precio, stock, fecha_importacion)
        self.fecha_vencimiento = fecha_vencimiento

    def get_fecha_vencimiento(self):
        return self.fecha_vencimiento

    def get_datos(self):
        return f"{super().get_datos()}    Fecha de Caducidad: {self.get_fecha_vencimiento()}"